app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -22,7 +22,10 @@
       <ImageView
         android:id="@+id/topic_thumbnail_image_view"
         android:layout_width="0dp"
-        android:layout_height="200dp"
+        android:layout_height="210dp"
+        android:paddingTop="22dp"
```

Comment:
This seems incorrect to me.

Reply:
There is no padding mentioned in the Mock. Height is mentioned in 202dp and it is corrected.

@mschanteltc do we need to add padding for the image over here.

Reply:
Somewhat, but the padding is negative. The top portion of the thumbnail image is overlapped by the tabs by 8 px.

Reply:
Okay, once tab is shown it will be covered. Can you specify what padding I should give.

Reply:
As per our discussion on mail, issue is with the image and there is no padding around the thumbnail image. Hence removed padding.